modules/apps/commerce/commerce-warehouse-web/src/main/resources/META-INF/resources/view_warehouse_items.jsp

```diff
@@ -39,10 +39,11 @@ if (Validator.isNotNull(backURL)) {
 				<aui:input name="redirect" type="hidden" value="<%= currentURL %>" />
 				<aui:input name="commerceInventoryWarehouseId" type="hidden" />
 				<aui:input name="commerceInventoryWarehouseItemId" type="hidden" />
+				<aui:input name="mvccVersion" type="hidden" />
```

Comment:
not sure about the position of this at this point, it should follow the parameters order?